alerting/provider/telegram/telegram.go

```diff
@@ -32,29 +32,29 @@ type AlertProvider struct {
 
 // Override is a configuration that may be prioritized over the default configuration
 type Override struct {
-	group string `yaml:"group"`
-	token string `yaml:"token"`
-	id    string `yaml:"id"`
+	Group string `yaml:"group"`
+	Token string `yaml:"token"`
+	ID    string `yaml:"id"`
 }
 
 // IsValid returns whether the provider's configuration is valid
 func (provider *AlertProvider) IsValid() bool {
-	if provider.ClientConfig == nil {
-		provider.ClientConfig = client.GetDefaultConfig()
-	}
-
-	registerGroups := make(map[string]bool)
-	for _, override := range provider.Overrides {
-		if len(override.group) == 0 {
-			return false
-		}
-		if _, ok := registerGroups[override.group]; ok {
-			return false
-		}
-		registerGroups[override.group] = true
-	}
-
-	return len(provider.Token) > 0 && len(provider.ID) > 0
+    if provider.ClientConfig == nil {
+        provider.ClientConfig = client.GetDefaultConfig()
+    }
+
+    registerGroups := make(map[string]bool)
+    for _, override := range provider.Overrides {
+        if len(override.Group) == 0 {
+            return false
+        }
+        if _, ok := registerGroups[override.Group]; ok {
+            return false
+        }
+        registerGroups[override.Group] = true
+    }
+
+    return len(provider.Token) > 0 && len(provider.ID) > 0
```

Comment:
Can you undo this change?

Reply:
The spaces?

Reply:
Yes, indentation in Go is with tabs, not spaces. It was previously indented with tabs, but your IDE must've automatically changed it.